instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -88,11 +91,31 @@ public String getProperty(String name) {
   /**
    * Returns a string-valued configuration property or {@code defaultValue} if a property with name
    * {@code name} has not been configured.
+   *
+   * @deprecated Use {@link #getString(String, String)} instead.
    */
+  @Deprecated
```

Comment:
I had to leave all those old methods and make them deprecated -- muzzle wouldn't let me remove them in this PR, I'll have to do that in the next one.